flow/raster_cache_unittests.cc

```diff
@@ -122,5 +124,29 @@ TEST(RasterCache, SweepsRemoveUnusedFrames) {
   ASSERT_FALSE(cache.Get(*picture, matrix).is_valid());
 }
 
+TEST(RasterCache, TryRasterizngOnlyOnce) {
+  size_t threshold = 1;
+  flutter::RasterCache cache(threshold);
+
+  SkMatrix matrix = SkMatrix::I();
+  // Test picture too large to successfully rasterize.
+  auto picture = SkPicture::MakePlaceholder(SkRect::MakeWH(2e12, 2e12));
+
+  sk_sp<SkColorSpace> srgb = SkColorSpace::MakeSRGB();
+  ASSERT_FALSE(cache.Prepare(NULL, picture.get(), matrix, srgb.get(), true,
```

Comment:
Nit: any particular reason of using `NULL` instead of `nullptr`? I think we generally prefer `nullptr` to `NULL`.